lib/matplotlib/cbook/__init__.py

```diff
@@ -1019,24 +1011,15 @@ def join(self, a, *args):
         self.clean()
 
     def joined(self, a, b):
-        """
-        Returns True if *a* and *b* are members of the same set.
-        """
+        """Returns True if *a* and *b* are members of the same set."""
         self.clean()
-
-        mapping = self._mapping
-        try:
-            return mapping[ref(a)] is mapping[ref(b)]
-        except KeyError:
-            return False
+        return self._mapping.get(ref(a), object()) is self._mapping.get(ref(b))
 
     def remove(self, a):
         self.clean()
-
-        mapping = self._mapping
-        seta = mapping.pop(ref(a), None)
-        if seta is not None:
-            seta.remove(ref(a))
+        set_a = self._mapping.pop(ref(a))
```

Comment:
I think you still need the second argument, None.  Otherwise you will risk a KeyError.